doc/develop/test/ztest.rst

```diff
@@ -138,6 +138,20 @@ it needs to report either a pass or fail.  For example::
 		ztest_run_test_suite(common);
 	}
 
+For tests that are under development, you can use the following macro at the
```

Comment:
I wouldn't limit this documentation to just tests under development. I would phrase it as the ability to skip tests based on any statically assertable condition. Then give the example you have.